package.json

```
@@ -11,7 +11,7 @@
     "test:watch": "jest --watch",
     "start": "grunt serve",
     "prettier": "prettier --write \"src/**/*.js\"",
-    "validate-prettier": "prettier --version; prettier --no-color --check \"src/**/*.js\""
+    "validate-prettier": "npm run prettier && prettier --version; prettier --no-color --check \"src/**/*.js\""
```

Shouldn't that be consistent?

```suggestion
    "validate-prettier": "npm run prettier && npm run prettier --version; npm run prettier --no-color --check \"src/**/*.js\""
```